<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::TargetMoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Microcode"/>
    <addaction name="actionLoad_Microcode"/>
    <addaction name="actionSave_MicroCode"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Configuration"/>
    <addaction name="actionLoad_Configuration"/>
    <addaction name="actionSave_Configuration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionEdit_Configuration"/>
   </widget>
   <widget class="QMenu" name="menuMicroprogram">
    <property name="title">
     <string>Microprogram</string>
    </property>
    <addaction name="actionAdd_Instruction"/>
    <addaction name="separator"/>
    <addaction name="actionGenerate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuMicroprogram"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Microcode">
   <property name="text">
    <string>Load Microcode...</string>
   </property>
  </action>
  <action name="actionSave_MicroCode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save MicroCode...</string>
   </property>
  </action>
  <action name="actionLoad_Configuration">
   <property name="text">
    <string>Load Configuration...</string>
   </property>
  </action>
  <action name="actionSave_Configuration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Configuration...</string>
   </property>
  </action>
  <action name="actionEdit_Configuration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edit Configuration...</string>
   </property>
  </action>
  <action name="actionNew_Microcode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Microcode</string>
   </property>
  </action>
  <action name="actionNew_Configuration">
   <property name="text">
    <string>New Configuration</string>
   </property>
  </action>
  <action name="actionAdd_Instruction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Instruction...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate...</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="text">
    <string>Binary</string>
   </property>
   <property name="toolTip">
    <string>Binary</string>
   </property>
  </action>
  <action name="actionHexadecimal">
   <property name="text">
    <string>Hexadecimal</string>
   </property>
   <property name="toolTip">
    <string>Hexadecimal</string>
   </property>
  </action>
  <action name="actionDecimal">
   <property name="text">
    <string>Decimal</string>
   </property>
   <property name="toolTip">
    <string>Decimal</string>
   </property>
  </action>
  <action name="actionAdd_Instruction_Context">
   <property name="text">
    <string>Add Instruction</string>
   </property>
  </action>
  <action name="actionDelete_Instruction_Context">
   <property name="text">
    <string>Delete Instruction</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>AddInstruction()</slot>
  <slot>EditConfiguration()</slot>
 </slots>
</ui>
